src-tauri/src/ron_utils.rs

```diff
@@ -0,0 +1,11 @@
+use crate::config::config;
+
+pub struct RonUtils {}
+
+impl RonUtils {
+    pub fn load_config() -> config {
+        let ron_string = std::fs::read_to_string("config.ron").unwrap();
+        let result: config = ron::de::from_str(&ron_string).unwrap();
+        result
+    }
+}
```

Comment:
tfuj, tohle je oop-brained narovnávák na ohejbák a nemá tu co dělat. Tenhle config loading by měl bejt v Configu, a tohle by nemělo existovat, zejména ta RonUtils struktura :D 

btw, pokud chceš unit struct, tak lepší způsub jak to napsat je:

```rs
pub struct Neco;
```

tak je to správně